pyrene/src/components/Tooltip/Tooltip.tsx

```diff
@@ -23,7 +23,7 @@ export interface TooltipProps {
   /**
    * Sets the label displayed inside of the tooltip.
    */
-  label: React.ReactNode;
+  label: React.ReactNode | string;
```

Comment:
Doesn't `ReactNode` already include `string`?

Reply:
@cyrilwanner you are right. Thanks !
![Screenshot 2021-11-18 at 13 20 30](https://user-images.githubusercontent.com/6510794/142414440-bcf6f7e5-410c-4ae6-bea6-47be141b11b5.png)
!